hack/local-up-cluster.sh

```diff
@@ -968,6 +968,28 @@ function create_storage_class {
     fi
 }
 
+function create_csidriver {
+    YAML_FILE=${KUBE_ROOT}/cluster/addons/storage-crds/csidriver.yaml
+
+    if [ -e $YAML_FILE ]; then
+        echo "Create CSIDriver CRD"
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create -f $YAML_FILE
+    else
+        echo "No csidriver available."
+    fi
+}
+
+function create_csinodeinfo {
+    YAML_FILE=${KUBE_ROOT}/cluster/addons/storage-crds/csinodeinfo.yaml
+
+    if [ -e $YAML_FILE ]; then
+        echo "Create CSINodeInfo CRD"
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create -f $YAML_FILE
+    else
+        echo "No csinodeinfo available."
+    fi
+}
+
```

Comment:
Alternative: we could unify this into a `create_crd` function that takes a YAML file as parameter.

Reply:
Sure